neuralmonkey/logging.py

```diff
@@ -20,6 +20,8 @@ class Logging(object):
         os.environ.get("NEURALMONKEY_DEBUG_ENABLE", "none")]  # type: List[str]
     debug_disabled = [
         os.environ.get("NEURALMONKEY_DEBUG_DISABLE", "")]  # type: List[str]
+    strict_mode = [
+        os.environ.get("NEURALMONKEY_STRICT")]  # type: List[str]
```

Comment:
proč je to seznam? debug_{enabled,disabled} jsou seznamy labelů, ale tohle by měl bejt jen string a měl by se testovat jako boolean.